Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
I would say that I prefer ExpressibleByIntegerLiteral over the overloading of the `+` operator.
Motivation: having a `+` operator between two heterogeneous types is more mysterious than saying that an `Int` is automatically also a valid `Value` type. BUT: the thing we should discuss to me is if we want that 200 is equivalent to `.scalable(200)` or `.fixed(200)`